csharp/tests/Integration/GetAndSet.cs

```diff
@@ -2,19 +2,12 @@
  * Copyright GLIDE-for-Redis Project Contributors - SPDX Identifier: Apache-2.0
  */
 
-namespace tests;
+namespace tests.Integration;
 
 using Glide;
 
-// TODO - need to start a new redis server for each test?
-public class AsyncClientTests
+public class GetAndSet
 {
-    [OneTimeSetUp]
-    public void Setup()
-    {
-        Glide.Logger.SetLoggerConfig(Glide.Level.Info);
```

Comment:
why was this removed?

Reply:
Do we need that? Logs produce too much noise and aren't helpful.
If you need logs, I'll activate them, but in `IntegrationTestBase::SetUp`

Reply:
we use info in tests by default, in order to get decent logs on our tests. they're loud, but it helps for failures in CI.